docker/workspace-build/Dockerfile

```diff
@@ -107,6 +102,11 @@ COPY ./ros "${ANGEL_WORKSPACE_DIR}/src"
 COPY docker/workspace_build.sh "${ANGEL_WORKSPACE_DIR}"/
 RUN "${ANGEL_WORKSPACE_DIR}/workspace_build.sh"
 
+# For demo ui
+RUN curl -sL https://deb.nodesource.com/setup_14.x | bash - 
+RUN apt-get install -y nodejs && node -v && npm -v
```

Comment:
These top two were fine in the previous location next to the other system package installs (e.g. we don't want to redo these two steps every time we update the ROS workspace). The below build *does* make sense to be at this stage of the build.

Another thing to consider is adding this build to the workspace build script.